parlai/core/torch_agent.py

```diff
@@ -1394,7 +1398,10 @@ def _set_label_vec(self, obs, add_start, add_end, truncate):
 
         elif label_type + '_vec' in obs:
             # check truncation of pre-computed vector
+            label_length = len(obs[label_type + '_vec'])
             truncated_vec = self._check_truncate(obs[label_type + '_vec'], truncate)
+            obs.force_set('ori_label_length', label_length)
```

Comment:
please use long, verbose names for these. there's no reason not to

Reply:
Also a bit wondering about whether this will hit the fan in the case of a cached text_vec. I don't think we do use caching anymore but this kinda looks like it.